addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1198,6 +1198,7 @@ body.editor_enable.editor_has_snippets {
                 width: $o-we-sidebar-content-field-colorpicker-size;
                 height: $o-we-sidebar-content-field-colorpicker-size;
                 border: $o-we-sidebar-content-field-border-width solid $o-we-sidebar-content-field-border-color;
+                cursor: pointer;
```

Comment:
@detrouxdev This should be a fix in stable IMO (14.0 ?)

Reply:
I agree. Do I extract that line from the commit and make another branch with it in 14.0? (And keep the same task for it?)

Reply:
Yes of course :+1: